tests/testing_config/custom_components/test/select.py

```diff
@@ -0,0 +1,55 @@
+"""
+Provide a mock select platform.
+
+Call init before using it in your tests to ensure clean test data.
+"""
+from tests.common import MockEntity
+
+UNIQUE_SELECT_1 = "unique_select_1"
+UNIQUE_SELECT_2 = "unique_select_2"
+
+ENTITIES = []
+
+
+class MockSelectEntity(MockEntity):
```

Comment:
This should inherit the SelectEntity class too.
```suggestion
class MockSelectEntity(MockEntity, SelectEntity):
```